node/utils/region.ts

```diff
@@ -0,0 +1,4 @@
+export function isBase64(str: string | undefined) {
+    const base64Regex = /^[A-Za-z0-9+/=]+$/;
```

Comment:
This regex validates if there's something on it but as far as I remember it will pass values like `a` and stuffs like that, should this pass as well?
Isn't it better to validate if there's an value and if it can be decoded or something like that?